hadoop-project/pom.xml

```diff
@@ -210,6 +210,8 @@
     <openssl-wildfly.version>1.0.7.Final</openssl-wildfly.version>
     <jsonschema2pojo.version>1.0.2</jsonschema2pojo.version>
     <woodstox.version>5.3.0</woodstox.version>
+    <json-smart.version>2.4.2</json-smart.version>
+    <nimbus-jose-jwt.version>9.8</nimbus-jose-jwt.version>
```

Comment:
Guess 9.8.1 is also available? Or did I check it wrong, If so we can choose that?
Otherwise the build seems happy,

Reply:
That's correct, 9.8.1 seems latest. Maven repository showed highest usage for 9.8 and 9.8.1 is quite recent, hence I thought of using 9.8. However, I think we can use 9.8.1 as things look good with 9.8 so there should not be much difference with 9.8.1 either.